client/src/pages/Login.js

```diff
@@ -1,7 +1,106 @@
-import { Box } from "@material-ui/core";
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState, useEffect } from 'react'
```

Comment:
```suggestion
import React from 'react'
```
These two imports are not being used

Reply:
for sure, I will remove it